openmdao/docs/openmdao_book/advanced_user_guide/jax_derivatives/partial_derivs_explicit.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>I sort of take back my suggestion to change away from "primal" if it's borrowing from other terminology and is used throughout.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2933/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>